pkgs/development/tools/build-managers/meson/setup-hook.sh

```diff
@@ -20,3 +20,13 @@ if [ -z "$dontUseMesonConfigure" -a -z "$configurePhase" ]; then
     setOutputFlags=
     configurePhase=mesonConfigurePhase
 fi
+
+mesonPreFixupPhase() {
+    for f in $(find . -name .nix-dont-strip); do
+	echo "Found $f, setting dontStrip"
+	export dontStrip=1
+	rm $f
```

Comment:
indentation mismatch

Reply:
Can't get Emacs to indent shell scripts properly...